src/frontend/Felles/Toast/Toast.tsx

```diff
@@ -36,6 +36,12 @@ export const Toast: React.FC = () => {
         case null:
         case undefined:
             return null;
+        case EToast.REDIRECT_ANNEN_RELASJON_FEILET:
```

Comment:
Er vi sikre på at den ikke er mer synlig dersom den opptrer midt på skjermen?

Reply:
Nei, flytter til midten. Men stusser litt på 16% bredde som default - virker litt smalt for så mye tekst som et par har ...